x/tokenfactory/keeper/bankactions.go

```diff
@@ -68,6 +72,28 @@ func (k Keeper) forceTransfer(ctx sdk.Context, amount sdk.Coin, fromAddr string,
 		return err
 	}
 
+	fromAcc, err := sdk.AccAddressFromBech32(fromAddr)
+	if err != nil {
+		return err
+	}
+
+	sortedPermAddrs := make([]string, 0, len(k.permAddrs))
+	for moduleName := range k.permAddrs {
+		sortedPermAddrs = append(sortedPermAddrs, moduleName)
+	}
+	sort.Strings(sortedPermAddrs)
```

Comment:
is this deterministic? I know sort.Slice is not, I assume it's the same here?

If you can't sort, iter through all, don't break early. Then if an error, return after the for loop